### PR TITLE
Add highlights for March 11, 2022

### PR DESCRIPTION
**Pull request by gianfranco-l**
Add highlights for March 11, 2022

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by PrashantR30** [comment]
Hii @gianfranco-l. There is no additions from Rainbow side for this week.

File: src/content/changes/highlights/2022-03-11.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+## Apps
+
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2022-03-11.md:14]:
```suggestion
[kong-app](https://github.com/giantswarm/kong-app) [v2.7.2](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#272---2022-03-10) fixes an issue with permissions in the CRD installation.

```

**Comment by gianfranco-l** [comment]
@alex-dabija @cornelius-keller anything to announce this week from your side?
@weatherhog  anything else from atlas and cabbage? please confirm

**Comment by weatherhog** [comment]
@gianfranco-l thanks a lot buddy. Nothing else to add from atlas or cabbage.

**Review by ubergesundheit — APPROVED** [review]

**Comment by alex-dabija** [comment]
Nothing from team Phoenix.

**Comment by ljakimczuk** [comment]
From Honeybadger:
* the `Authentication for the Management API` doc has been updated with an example of how to create `kubeconfig` for programmatic access
* the `kubectl gs get catalog <name>` command's output has been extended with app description column

**Pull request merged**